design-toolkit/components/src/components/avatar/index.tsx

```diff
@@ -0,0 +1,87 @@
+/*
+ * Copyright 2025 Hypergiant Galactic Systems Inc. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+
+'use client';
+import { Person } from '@/icons/person';
+import { cn } from '@/lib/utils';
+import * as AvatarPrimitive from '@radix-ui/react-avatar';
+import { type VariantProps, cva } from 'cva';
+import type React from 'react';
+
+const avatarStyles = cva(
+  'flex items-center justify-center overflow-hidden rounded-full bg-surface-overlay text-default-dark',
+  {
+    variants: {
+      size: {
+        medium: 'size-[32px] **:[svg]:size-xl',
+        small: 'size-xl **:[svg]:size-l',
+      },
+    },
+    defaultVariants: {
+      size: 'medium',
+    },
+  },
+);
+
+export interface AvatarProps
+  extends React.ComponentProps<typeof AvatarPrimitive.Image>,
+    VariantProps<typeof avatarStyles> {
+  fallback?: React.ReactNode;
+  fallbackDelay?: number;
+  source?: string | React.ReactNode;
+}
+
+export const Avatar = ({
+  className,
+  children,
+  fallback,
+  fallbackDelay,
+  source,
+  size,
+  ...props
+}: AvatarProps) => (
+  <div
+    className={cn([
```

Comment:
If a `cn` call is completely static e.g. no dynamic values, can it be hoisted outside of the react component?